cmake/Summary.cmake

```diff
@@ -95,11 +95,8 @@ function (caffe2_print_configuration_summary)
     message(STATUS "    LMDB version        : ${LMDB_VERSION}")
   endif()
   message(STATUS "  USE_METAL             : ${USE_METAL}")
-  message(STATUS "  USE_MKL               : ${USE_MKL}")
-  if(${USE_MKL})
-    message(STATUS "    USE_MKLML           : ${USE_MKLML}")
-    message(STATUS "    USE_IDEEP           : ${USE_IDEEP}")
-  endif()
+  message(STATUS "  USE_MKL               : ${CAFFE2_USE_MKL}")
+  message(STATUS "  USE_MKLDNN            : ${CAFFE2_USE_IDEEP}")
```

Comment:
There should really be a 1-to-1 between the message and variable so let's call these `USE_MKL` and `USE_MKLDNN` if that works.